R/phylo_check.R

```diff
@@ -0,0 +1,133 @@
+#' Check phylogeny tip names
+#'
+#' A function to check the list of tip names in a phylogeny against a vector of
+#' taxon names, and if desired, to trim the phylogeny to only include taxon
+#' names within the vector.
+#'
+#' @param tree \code{phylo}. A phylo object. Phylogenies can be read into R from
+#' .txt or .tree files containing the Newick formatted tree using
+#' \code{ape::read.tree}.
+#' @param list \code{character}. A vector of taxon names. Binomials can be
+#' separated with either a space or an underscore. The list should not contain
+#' any punctuation.
+#' @param out \code{character}. Determine whether to return either the counts
+#' of taxa included and not included in the tree ("counts", the default), a
+#' \code{dataframe} describing which taxa are included or not included in the
+#' tree ("table"), or the phylogeny trimmed to only include taxa in the provided
+#' list ("tree").
+#' @return If out = "counts", a summary table containing the number of taxa in
+#' the list but not the tree, in the tree but not the list, and in both. If out
+#'  = "table", a \code{dataframe} describing whether taxon names are present in
+#' the list and/or the tree. If out = "tree", the input phylogeny trimmed to
+#' only include the tips present in the list; you can save your new tree using
+#' \code{ape::write.tree}.
```

Comment:
Again, I think this information about `write.tree` should go in a details section. And also make sure to link to the ape documentation.